.vscode/launch.json

```diff
@@ -70,45 +48,5 @@
                 "${workspaceRoot}"
             ]
         },
-        {
-            "name": "RSpec - active spec file only",
```

Comment:
I will use them eventually so would like them there so I don't have to go find them later.